src/builder/command.rs

```diff
@@ -708,6 +708,54 @@ impl<'help> App<'help> {
         c.print()
     }
 
+    /// Returns the help message (`-h`).
+    ///
+    /// # Examples
+    ///
+    /// ```rust
+    /// # use clap::Command;
+    /// let mut cmd = Command::new("myprog");
+    /// let help: String = cmd.get_help();
+    /// print!("{}",help);
+    /// ```
+    pub fn get_help(&mut self) -> String {
```

Comment:
As mentioned in #3873, there is a way to get an in-memory version.  If you are wanting to move forward with this, we'd need to first decide in the issue which way we should go.

If we do move forward with this
- We've been using the `render` prefix to connote there is work to do rather than being a dumb getter
- We shouldn't include color in the output